api-report/runtime-definitions.api.md

```diff
@@ -215,6 +215,20 @@ export interface IGarbageCollectionData {
     };
 }
 
+// @public
+export interface IGarbageCollectionNodeData {
+    outboundRoutes: string[];
+    unreferencedTimestampMs?: number;
+}
+
+// @public
+export interface IGarbageCollectionState {
```

Comment:
safe feedback as in another PR - should this be type GarbageCollectionState = Map<string,  IGarbageCollectionNodeData>?

Reply:
An object works well with JSON so it's easier to serialize it during summary. For map, we will have to iterate it or convert to array before serializing. Although, it may be worth it. Let me give it a try.

Reply:
I tried map and it works just fine. The only thing is we have to convert from map to array before serializing it and from array to map when deserializing. That creates a new type - an array or arrays that we store in the summary. I don't see any obvious advantage of doing this so would prefer to keep the object itself.